cdc/entry/mounter_group.go

```diff
@@ -62,13 +67,9 @@ func NewMounterGroup(
 	if workerNum <= 0 {
 		workerNum = defaultMounterWorkerNum
 	}
-	inputCh := make([]chan *model.PolymorphicEvent, workerNum)
-	for i := 0; i < workerNum; i++ {
-		inputCh[i] = make(chan *model.PolymorphicEvent, defaultInputChanSize)
-	}
 	return &mounterGroup{
 		schemaStorage:  schemaStorage,
-		inputCh:        inputCh,
```

Comment:
In this PR, all events have been counted into memory quota before being pushed into mouter group. So it won't cause memory problems.

Reply:
In this PR mounter group shares the same memory quota with table sink worker, considering the following optimization, is it necessary
- Use independent memory quota for mounter group, in case of some deadlock problem
- Update default value for memory quota, since after this PR more memory is tracked in quota.

Reply:
For every table sink task (and redo log task, too), the first entry can always get memory even if quota is full. So deadlocks shouldn't appear.